### DISCUSSION TITLE
Translation: korean

### DISCUSSION THREAD

**Discussion by xnousnow** [rxhanson/Rectangle#1054] [General]
I'm native, I'd like to translate korean. I tryed but don't know how the translation works.

When translation is done, I'll open a PR. I'll appreciate if you accept me.

**Comment by rxhanson** [comment]
Sure, I'll accept a PR for it. Here is the file to modify: https://github.com/rxhanson/Rectangle/blob/master/Rectangle/ko.lproj/Main.strings
